src/resources/postcss/category-colors/frontend-category.pcss

```diff
@@ -19,15 +19,15 @@
 	/* Featured event styling */
 	.tribe-events-calendar-day__event--featured {
 		.tribe-events-calendar-day__event-datetime-featured-icon-svg {
-			color: var(--tec-color-category-primary, var(--global-palette3));
+			color: var(--tec-color-category-primary, var(--tec-color-border-default));
```

Comment:
Is `--tec-color-border-default` already defined somewhere? Just asking to be sure 🙂

Reply:
Yes, `--tec-color-border-default` is set in Common/Utilities.